wrapanapi/systems/ec2.py

```diff
@@ -499,17 +473,17 @@ def find_vms(self, name=None, id=None, filters=None, hide_deleted=True):
                 "You must select one of these search methods: name, id, or filters")
 
         if id:
-            kwargs = {'instance_ids': [id]}
+            kwargs = {'InstanceIds': [id]}
         elif filters:
-            kwargs = {'filters': filters}
+            kwargs = {'Filters': filters}
         elif name:
             # Quick validation that the instance name isn't actually an ID
             pattern = re.compile(r'^i-\w{8,17}$')
             if pattern.match(name):
                 # Switch to using the id search method
-                kwargs = {'instance_ids': [name]}
+                kwargs = {'InstanceIds': [name]}
             else:
-                kwargs = {'filters': {'tag:Name': name}}
+                kwargs = {'Filters': [{'Name': 'tag:Name', 'Value': name}]}
```

Comment:
This doesn't appear to be working correctly with a name filter passed.

```
In [14]: [i.name for i in e.mgmt.find_vms(name='test-pr-iagg')]
returning cached mgmt class for 'ec2west'
Out[14]: 
['test-pr-iagg',
 'Fedora_26-ssa',
 'sbulage_test_instance',
 'cu-24x7',
 'Ubuntu-14-ssa',
 'sbulage_ssh_dnd',
 'smartstate',
 'windows-server-2012-r2_ssa',
 'smartstate',
 'cu-9-5',
 'smartstate',
 'windows-server-2016_ssa',
 'sbulage_instance_check_user',
 'dmisharo']
```

Reply:
Fixed.

Reply:
I'm still seeing this behavior on https://github.com/ManageIQ/wrapanapi/commit/a479e18feb92787d5f4d191a661e42ee2e740983

```
In [5]: [i.name for i in e.mgmt.find_vms(name='test-pr-iagg')]
returning cached mgmt class for 'ec2west'
Out[5]: 
['Fedora_26-ssa',
 'sbulage_test_instance',
 'cu-24x7',
 'Ubuntu-14-ssa',
 'sbulage_ssh_dnd',
 'smartstate',
 'windows-server-2012-r2_ssa',
 'test-rnnbSvblEn',
 'smartstate',
 'cu-9-5',
 'smartstate',
 'windows-server-2016_ssa',
 'sbulage_instance_check_user',
 'dmisharo']
```

Reply:
Should be fine now....There were multiple issues for this method.